openapi/edocs/Ballerina.toml

```
@@ -0,0 +1,11 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Content & Files/Documents", "Cost/Freemium"]
+org = "ballerinax"
+name = "edocs"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.0.0-SNAPSHOT"
```

Refer https://github.com/ballerina-platform/ballerina-extended-library/issues/134
```suggestion
version = "1.0.0"
```